services/httpd/config.go

```diff
@@ -44,6 +44,7 @@ type Config struct {
 	PprofAuthEnabled        bool           `toml:"pprof-auth-enabled"`
 	DebugPprofEnabled       bool           `toml:"debug-pprof-enabled"`
 	PingAuthEnabled         bool           `toml:"ping-auth-enabled"`
+	HTTPHeaders             [][2]string    `toml:"http-headers"`
```

Comment:
`[][2]string` feels a bit cumbersome. Maybe a map type would be a better fit here.

Reply:
agreed!